source/Jobbr.Server.ForkedExecution.Tests/JobExecutorOnPlanChangedTests.cs

```diff
@@ -81,7 +81,7 @@ public void ContainsAItemInPlan_GetsAdditional_ExecutesMultipleJobs()
             executor.OnPlanChanged(new List<PlannedJobRun>(new[] { fakeJobRun1.PlannedJobRun, fakeJobRun2.PlannedJobRun }));
 
             // Wait
-            var didStart2Jobs = this.ProgressChannelStore.WaitForStatusUpdate(updatesFromAllJobs => updatesFromAllJobs.Count == 2 && updatesFromAllJobs.All(kvp => kvp.Value.Contains(JobRunStates.Starting)), 5000);
+            var didStart2Jobs = this.ProgressChannelStore.WaitForStatusUpdate(updatesFromAllJobs => updatesFromAllJobs.Count == 2 && updatesFromAllJobs.All(kvp => kvp.Value.Contains(JobRunStates.Starting) && kvp.Value.Count == 1), 5000);
```

Comment:
The test says, fakeJobRun1 is not runned a second time, but there are two States Starting for fakeJobRun1, because it started twice